src/features/team-load-team.js

```diff
@@ -0,0 +1,91 @@
+var debug = require('../debug'),
+    vars = require('../vars'),
+    loadChannel = require('./team-load-channel');
+
+var loadTeam = module.exports = function() {
+    debug.log("Loading team data");
+    var theTeam = (window.location.pathname).replace("/team/", "");
+    theTeam = theTeam.replace("/event/", "");
+
+    Twitch.api.get("/api/team/"+theTeam+"/all_channels.json")
+    .done(function(d) {
+        vars.jsnTeam = d.channels;
+        createButtons();
+        setTimeout(loadTeam, 60000);
+        
+        if(vars.teamFirstLoad == 1) {
+            loadChannel(vars.teamCurrentChannel);
+            vars.teamFirstLoad = 0;
+        }
+    })
+    .fail(function(data) {
+        debug.log("team load failed");
+        setTimeout(loadTeam, 10000);
+    });
+
+    var createButtons = function() {
+        $("#team_member_list").empty();
+
+        vars.jsnTeam.forEach(function(a) {
+            var chanName = (a.channel.name).toLowerCase(),
+                dispName = a.channel.display_name,
+                chanImgUrl = a.channel.image.size50,
+                chanGame = a.channel.meta_game,
+                chanStatus = a.channel.status,
+                chanViewers = a.channel.current_viewers,
+                newDiv = $("<div>", {"id":"channel_"+chanName, "class":"member", "title":dispName+" is offline"});
+            
+            newDiv.click(function(e) {
+                loadChannel(chanName); 
+            });
+            
+            if(chanStatus == "live") {
+                newDiv.addClass("live");
+
+                var ttTime = new Date().getTime(),
+                    ttImgUrl = "http://static-cdn.jtvnw.net/previews-ttv/live_user_"+chanName+"-320x200.jpg?"+ttTime;
```

Comment:
No problem on using //

As far as hardcoding the preview image link: The team API doesn't return preview images in the channel objects. So instead of making an additional API call for each preview image I thought it would just be better to use the URL because it's defined in the streams API as a template URL anyways.

But I can make a call to /streams for the preview URL if you think it would be better.